rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2504,6 +2504,15 @@ public long getNextPegoutCreationBlockNumber() {
         return activations.isActive(RSKIP271) ? provider.getNextPegoutHeight().orElse(0L) : 0L;
     }
 
+    public int getQueuedPegoutsCount() {
+        try {
+            return provider.getReleaseRequestQueueSize();
+        } catch (IOException e) {
+            logger.error("Unexpected error accessing storage while attempting to get the ReleaseRequestQueueSize", e);
+        }
+        return 0;
+    }
```

Comment:
In the Bridge we should check for RSKIP activation like you had before. Since this method did not exist prior to this HF.

Reply:
Resolved